scripts/check_style.sh

```diff
@@ -3,54 +3,54 @@
 ERROR_LOG="$(mktemp -t check_style_XXXXXX.log)"
 
 EXCLUDE_FILES=(
-  "libsolutil/picosha2.h"
-  "test/libsolutil/UTF8.cpp"
+    "libsolutil/picosha2.h"
+    "test/libsolutil/UTF8.cpp"
 )
 EXCLUDE_FILES_JOINED=$(printf "%s\|" "${EXCLUDE_FILES[@]}")
 EXCLUDE_FILES_JOINED=${EXCLUDE_FILES_JOINED%??}
 
 (
 REPO_ROOT="$(dirname "$0")"/..
-cd $REPO_ROOT
+cd "$REPO_ROOT" || exit 1
```

Comment:
Switching to `set -e` would be better but I did not have enough time to test the script with it properly so I decided to go with this for now.